lib/oli/accounts.ex

```diff
@@ -371,6 +371,33 @@ defmodule Oli.Accounts do
   """
   def get_author!(id), do: Repo.get!(Author, id)
 
+  @doc """
+  Gets a single author with the count of the communities for wich the author is an admin.
+
+  ## Examples
+      iex> get_author_with_community_admin_count(1)
+      %Author{community_admin_count: 1}
+
+      iex> get_author_with_community_admin_count(456)
+      nil
+
+  """
+  def get_author_with_community_admin_count(id) do
```

Comment:
Can you clean this function / query up a bit? I think it does capture the heart of what he wanted.